x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/EnrichProcessorFactory.java

```diff
@@ -67,6 +67,11 @@ public Processor create(Map<String, Processor.Factory> processorFactories, Strin
             throw ConfigurationUtils.newConfigurationException(TYPE, tag, "max_matches", "should be between 1 and 128");
         }
 
+        if (EnrichPolicy.RANGE_MATCH_TYPES.contains(policyType)) {
+            // range match uses the same processor as simple match
```

Comment:
Maybe extend the comment here by including that a term query on a range field is translated into a range query with the values as lower and upper value. If multiple values are provided then each value is translates into a range query and that is then combined into a boolean query with should clauses, which means that on of the provided values need to fall into the range of the range field.